app/components/LoginForm.js

```diff
@@ -1,53 +1,41 @@
 // @flow
 
 import React, { Component } from 'react';
-import { View, StyleSheet, TextInput, Button } from 'react-native';
+import { View, StyleSheet, Button } from 'react-native';
+import { authorize, prefetchConfiguration } from 'react-native-app-auth';
+import Config from 'react-native-config';
+import { storeData as session } from '../utils/Storage';
 
-type Props = {
-  loginUser: any,
-};
+type Props = {};
+
+type State = {};
```

Comment:
Just for sake of understanding, why are you stating the props and state as global vars?